src/components/ResourceCard.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+const ResourceCard = (props) => {
+  const { title, resourceimg, description, resourceurl, tag } = props
```

Comment:
I think we need to make a change to the props names, for consistency across the code base. The props should change to `resourceImage` and `resourceUrl`. Obviously this will change in the `LearningResources.js` page as well. I hate to be a stickler, but we will need to refactor this.

Reply:
Refactored ResourceCard.js props to ```resourceImage``` and ```resourceUrl```

Updated LearningResources.js props to reflect the new component props